packages/atri-app-core/src/canvasMachine.ts

```diff
@@ -106,6 +108,13 @@ type PROGRAMTIC_SELECT_EVENT = {
   type: typeof PROGRAMTIC_SELECT;
   id: string;
 };
+type DROP_ZONE_CREATED_EVENT = {
+  type: typeof DROP_ZONE_CREATED;
+  id: string;
+};
+type DROP_ZONE_DESTROYED_EVENT = {
+  type: typeof DROP_ZONE_DESTROYED;
+};
```

Comment:
We don't need these two `type DROP_ZONE_CREATED_EVENT` and `type DROP_ZONE_DESTROYED_EVENT` because it's not an event that's received by the `canvasMachine`. These two are just utility events.

Reply:
Done